src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedToObject = {};
+  const splitedSourceString = sourceString.split(';');
+
+  const arrayOfProperties = splitedSourceString.map(string =>
+    string.trim().split(':'));
```

Comment:
Як на мене лишній раз ділиш, можна одразу масив записати::
```suggestion
  const arrayOfProperties = sourceString.split(';').map(string =>
    string.trim().split(':'));;
```

Reply:
хэх, ну можно и так, главное то читабельность, но на что не пойдешь ради апрува :D Сегодня сделаю

Reply:
Ты был прав, так мне нравиться больше )